flink-table/flink-table-planner/src/main/java/org/apache/flink/table/planner/plan/rules/logical/JoinToMultiJoinForReorderRule.java

```diff
@@ -99,24 +99,24 @@
  * @see CoreRules#JOIN_TO_MULTI_JOIN
  */
 @Value.Enclosing
-public class FlinkJoinToMultiJoinRule extends RelRule<FlinkJoinToMultiJoinRule.Config>
+public class JoinToMultiJoinForReorderRule extends RelRule<JoinToMultiJoinForReorderRule.Config>
         implements TransformationRule {
 
-    public static final FlinkJoinToMultiJoinRule INSTANCE =
-            FlinkJoinToMultiJoinRule.Config.DEFAULT.toRule();
+    public static final JoinToMultiJoinForReorderRule INSTANCE =
+            JoinToMultiJoinForReorderRule.Config.DEFAULT.toRule();
 
-    /** Creates a JoinToMultiJoinRule. */
-    public FlinkJoinToMultiJoinRule(Config config) {
+    /** Creates a JoinToMultiJoinForReorderRule. */
+    public JoinToMultiJoinForReorderRule(Config config) {
         super(config);
     }
 
     @Deprecated // to be removed before 2.0
-    public FlinkJoinToMultiJoinRule(Class<? extends Join> clazz) {
+    public JoinToMultiJoinForReorderRule(Class<? extends Join> clazz) {
```

Comment:
if this constructor is not used in our code base, feel free to remove unused code.